src/test/java/net/bhardy/bizzo/billing/PolicyBuilderTest.java

```diff
@@ -33,233 +33,197 @@
+                .daily()
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-        List<LocalDate> nextBills = policy.upcomingDueDates(oneSaturday, 2);
-
-        LocalDate expected1 = LocalDate.of(2017, 8, 5);
-        LocalDate actual1 = nextBills.get(0);
-        assertEquals(expected1, actual1);
-        assertEquals(SATURDAY, actual1.getDayOfWeek());
+        List<LocalDate> nextBills = policy.upcomingDueDates(saturday5th, 2);
 
-        LocalDate expected2 = LocalDate.of(2017, 8, 6);
-        LocalDate actual2 = nextBills.get(1);
-        assertEquals(expected2, actual2);
-        assertEquals(SUNDAY, actual2.getDayOfWeek());
+        assertEquals(saturday5th, nextBills.get(0));
+        assertEquals(sunday6th, nextBills.get(1));
     }
 
     @Ignore  // this reveals an underlying bug we need to fix
     @Test
     public void upcomingDueDates_dailyFiltered() {
         BillingPolicy policy = BillingPolicy.builder()
-                .everyDay()
+                .daily()
                 .filter(not(daysOfWeek(SATURDAY, SUNDAY)))
                 .action(NEXT_DAY)
                 .build();
 
-        LocalDate oneSaturday = LocalDate.of(2017, 8, 5);
-
-        List<LocalDate> nextBills = policy.upcomingDueDates(oneSaturday, 2);
-
-        LocalDate expected1 = LocalDate.of(2017, 8, 7);
-        LocalDate actual1 = nextBills.get(0);
-        assertEquals(expected1, actual1);
-        assertEquals(MONDAY, actual1.getDayOfWeek());
+        List<LocalDate> nextBills = policy.upcomingDueDates(saturday5th, 2);
 
-        LocalDate expected2 = LocalDate.of(2017, 8, 8);
-        LocalDate actual2 = nextBills.get(1);
-        assertEquals(expected2, actual2);
-        assertEquals(TUESDAY, actual2.getDayOfWeek());
+        assertEquals(monday7th, nextBills.get(0));
+        assertEquals(tuesday8th, nextBills.get(1));
     }
 
     @Test
     public void upcomingDueDates_weeklySimple() {
+        LocalDate friday11th = friday4th.plusWeeks(1);
+
         BillingPolicy policy = BillingPolicy.builder()
-                .sameDayEveryWeek(FRIDAY)
+                .weeklyOnDay(FRIDAY)
                 .build();
+        List<LocalDate> nextBills = policy.upcomingDueDates(friday4th, 2);
 
-        LocalDate itsFriday = LocalDate.of(2017, 8, 4);
-        List<LocalDate> nextBills = policy.upcomingDueDates(itsFriday, 2);
-
-        LocalDate expected1 = LocalDate.of(2017, 8, 4);
-        LocalDate actual1 = nextBills.get(0);
-        assertEquals(expected1, actual1);
-        assertEquals(FRIDAY, actual1.getDayOfWeek());
-
-        LocalDate expected2 = LocalDate.of(2017, 8, 11);
-        LocalDate actual2 = nextBills.get(1);
-        assertEquals(expected2, actual2);
-        assertEquals(FRIDAY, actual2.getDayOfWeek());
+        assertEquals(friday4th, nextBills.get(0));
+        assertEquals(friday11th, nextBills.get(1));
     }
 
     @Test
     public void upcomingDueDates_weeklyFiltered() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)